easy-ui-react/src/Button/Button.test.tsx

```diff
@@ -34,4 +34,33 @@ describe("<Button />", () => {
     expect(screen.getByRole("button")).toBeInTheDocument();
     expect(screen.getByRole("img", { hidden: true })).toBeInTheDocument();
   });
+
+  it("should apply the primary color class", () => {
+    render(<Button />);
+    expect(screen.getByRole("button")).toHaveAttribute(
+      "class",
+      expect.stringContaining("Primary"),
```

Comment:
**nitpick:** i would probably opt for using the entire class names in these tests (and the `DropdownButton` tests) instead of a subset of the names. i think it's mildly confusing that it's uppercase. your call though

```suggestion
      expect.stringContaining("colorPrimary"),
```